Kronos-Shifts-Connector/Microsoft.Teams.Shifts.Integration/Microsoft.Teams.Shifts.Integration.API/Controllers/OpenShiftRequestController.cs

```diff
@@ -415,158 +408,159 @@ internal async Task ProcessOpenShiftsRequests(string isRequestFromLogicApp)
+                                            }
+                                            else
                                             {
-                                                var approvalHttpResponse = await approvalHttpClient.SendAsync(httpRequestMessage).ConfigureAwait(false);
-                                                if (approvalHttpResponse.IsSuccessStatusCode)
-                                                {
-                                                    this.telemetryClient.TrackTrace($"{Resource.ProcessOpenShiftsRequests} ShiftRequestId: {entityToUpdate.RowKey} KronosRequestId: {item.Id}");
-                                                }
-                                                else
-                                                {
-                                                    this.telemetryClient.TrackTrace($"{Resource.ProcessOpenShiftsRequests} -Error The service is acting on Open Shift Request: {entityToUpdate?.RowKey} which may have been approved");
-                                                }
+                                                this.telemetryClient.TrackTrace($"{Resource.ProcessOpenShiftsRequests} -Error The service is acting on Open Shift Request: {entityToUpdate?.RowKey} which may have been approved");
                                             }
                                         }
-                                        else
-                                        {
-                                            // Output to AppInsights logging.
-                                            this.telemetryClient.TrackTrace($"{Resource.ProcessOpenShiftsRequests} - Getting Open Shift for {entityToUpdate.TeamsOpenShiftId} results in {getOpenShiftResponse.Content.ToString()}");
-                                        }
+                                    }
+                                    else
+                                    {
+                                        // Output to AppInsights logging.
+                                        this.telemetryClient.TrackTrace($"{Resource.ProcessOpenShiftsRequests} - Getting Open Shift for {entityToUpdate.TeamsOpenShiftId} results in {getOpenShiftResponse.Content.ToString()}");
                                     }
                                 }
+                            }
 
-                                if ((item.StatusName == ApiConstants.Retract || item.StatusName == ApiConstants.Refused) && entityToUpdate.ShiftsStatus == ApiConstants.Pending)
-                                {
-                                    entityToUpdate.KronosStatus = item.StatusName;
-                                    entityToUpdate.ShiftsStatus = item.StatusName;
+                            if ((item.StatusName == ApiConstants.Retract || item.StatusName == ApiConstants.Refused) && entityToUpdate.ShiftsStatus == ApiConstants.Pending)
```

Comment:
same here, .equals for string comparsion

Reply:
Done